### PR TITLE
Perf tuning doc update with latest API

### PR DESCRIPTION
**Pull request by ybrnathan**
**Description**: Describe your changes.

**Motivation and Context**
- Why is this change required? What problem does it solve?
- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

File: docs/ONNX_Runtime_Perf_Tuning.md [hunk 1]
```diff
@@ -67,14 +79,19 @@ import onnxruntime as rt
 sess_options = rt.SessionOptions()
 
 sess_options.intra_op_num_threads=2
+<<<<<<< HEAD
```
Comment (pranavsharma) [inline docs/ONNX_Runtime_Perf_Tuning.md]: Can you fix the merge conflict markers?

Comment (ybrnathan) [reply docs/ONNX_Runtime_Perf_Tuning.md]: resolved.

File: docs/ONNX_Runtime_Perf_Tuning.md [hunk 2]
```diff
@@ -66,19 +76,19 @@ import onnxruntime as rt
...
+sess_options.graph_optimization_level = rt.GraphOptimizationLevel.ORT_ENABLE_ALL
 ```
 
 * Thread Count
-  * `sess_options.intra_op_num_threads=2` controls the number of threads to use to run the model
+  * `sess_options.intra_op_num_threads = 2` controls the number of threads to use to run the model
 * Sequential vs Parallel Execution
-  * `sess_options.execution_mode=rt.ExecutionMode.ORT_SEQUENTIAL` controls whether then operators in the graph should run sequentially or in parallel. Usually when a model has many branches, setting this option to false will provide better performance.
-  * When `sess_options.execution_mode=rt.ExecutionMode.ORT_PARALLEL`, you can set `sess_options.inter_op_num_threads` to control the
+  * `sess_options.execution_mode = rt.ExecutionMode.ORT_SEQUENTIAL` controls whether then operators in the graph should run sequentially or in parallel. Usually when a model has many branches, setting this option to false will provide better performance.
```
Comment (askhade) [inline docs/ONNX_Runtime_Perf_Tuning.md]: nit pick : whether "the" operators

**Pull request merged**